landlab/layers/eventlayers.py

```diff
@@ -0,0 +1,503 @@
+from __future__ import print_function
+
+import os
+import sys
+
+import numpy as np
+
+
+def _deposit_or_erode(layers, n_layers, dz):
+    """Add a new layer to the top of a stack.
+
+    Parameters
+    ----------
+    layers : ndarray of shape `(M, N)`
+        Array of layer thicknesses.
```

Comment:
Would be useful to say in comment what M and N index (presumably one is nodes, one is layers?)